client/app/dbaas/logs/detail/inputs/console/logs-inputs-console.html

```diff
@@ -1,7 +1,7 @@
 <div class="cui-page__content inputs-logs">
-    <oui-back-button class="back_button_inline_text"></oui-back-button>
-    <span class="oui-header_3" data-translate="inputs_logs_title" data-translate-values="{inputTitle: ctrl.input.data.info.title}"></span>
-    
+
+    <oui-back-button>{{ 'inputs_logs_title' | translate: {inputTitle: ctrl.input.data.info.title} }}</oui-back-button>
```

Comment:
Prefer using the `translate` directive instead of `{{`.

For instance:
```suggestion
    <oui-back-button>
        <span data-translate="inputs_logs_title"
              data-translate-values="{
                  'inputTitle:' ctrl.input.data.info.title
              }">
        </span>
    </oui-back-button>
```

Reply:
data-heading do not takes translate-params in account (inputTitle does not appears).

Reply:
You're right! According to the [documentation](https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/oui-angular/back-button), `heading|title` are deprecated.

But in my suggestion, I'm using a `<span>` element inside the `<oui-back-button>` element.

Hope its help you.

Reply:
Tested your suggestion, but it failed for me: nothing translated on screen, blank space.
I let mine.